doc/middleware-it-registratore-telematico/operation-modes/on-premise-installation.md

```diff
@@ -7,7 +7,8 @@ title: On-premise installation
 
 The illustration below shows the components available for the premise installation of the fiskaltrust.Middleware for the German market. For the operation modes available and the description of the components please refer to the [general part](../../general/operation-modes/operation-modes.md) .
 
-TBD
+![middleware-en](images/middleware-en-market-it.png)
```

Comment:
Docummento commerciale is not available for on premise instllations only as part of the cloudcashbox offering

Reply:
created the on-premise and the cloud solution blocks